ydb/core/tx/datashard/ut_common/datashard_ut_common.cpp

```diff
@@ -2515,4 +2556,123 @@ void SendViaPipeCache(
         /* viaActorSystem */ true);
 }
 
+void AddKeyQuery(
+    TEvDataShard::TEvRead& request,
+    const std::vector<ui32>& keys)
+{
+    // convertion is ugly, but for tests is OK
+    auto cells = ToCells(keys);
+    request.Keys.emplace_back(cells);
+}
+
+void AddFullRangeQuery(TEvDataShard::TEvRead& request) {
+    auto fromBuf = TSerializedCellVec::Serialize(TVector<TCell>());
+    auto toBuf = TSerializedCellVec::Serialize(TVector<TCell>());
+    request.Ranges.emplace_back(fromBuf, toBuf, true, true);
+}
+
+std::unique_ptr<TEvDataShard::TEvRead> GetBaseReadRequest(
+    const TTableId& tableId,
+    const NKikimrTxDataShard::TEvGetInfoResponse::TUserTable& userTable,
+    ui64 readId,
+    NKikimrDataEvents::EDataFormat format,
+    const TRowVersion& readVersion)
+{
+    auto request = std::make_unique<TEvDataShard::TEvRead>();
+    auto& record = request->Record;
+
+    record.SetReadId(readId);
+    record.MutableTableId()->SetOwnerId(tableId.PathId.OwnerId);
+    record.MutableTableId()->SetTableId(userTable.GetPathId());
```

Comment:
```suggestion
    record.MutableTableId()->SetTableId(tableId.PathId.LocalPathId);
```

`userTable` is not necessary.